docs/features/configuration.md

```diff
@@ -26,30 +31,39 @@ It takes a couple of seconds, but if you want to speed up your tests, you can di
 
 ## Customizing images
 
+!!! note
+    This approach is discouraged and deprecated, but is documented for completeness.
```

Comment:
Let's add something like "and deprecated in favour of [link to the image substitution feature]"

Reply:
Done

Reply:
the link is still missing I think